airbyte-integrations/connectors/source-bing-ads/metadata.yaml

```diff
@@ -51,6 +51,12 @@ data:
           Users will need to refresh the source schema and reset affected streams
           after upgrading.
         upgradeDeadline: "2023-12-11"
+      3.0.0:
```

Comment:
Why do we make it a breaking change? I thought the tenant_id was provided during the oauth flow and therefore should always be populated? Does that make sense to you? Can we validate that? If it is the case, I'm not sure it make sense bothering our users with a breaking change. The only caveat to that is for users creating the connector through the API but I assume they also provide the tenant_id else the connector does not work